src/utils.js

```diff
@@ -175,37 +190,47 @@ function isFile(p) {
   return new Promise((resolve, _) => {
     fs.stat(p, (err, stats) => {
       if (err || !stats) {
-        return resolve(false)
+        return resolve(false);
       }
-      resolve(stats.isFile())
-    })
-  })
+      return resolve(stats.isFile());
+    });
+  });
```

Comment:
This function should be refactored per the same pattern as `isDirectory` above, since it also uses an `fs` method. It's the only other such function in the repo, as far as I can tell.